src/components/StaffList/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+import Autosuggest from 'react-autosuggest';
+import DiscoveryDataApiService from '../../data/services/DiscoveryDataApiService';
```

Comment:
nit: we usually separate relative imports from absolute imports

Reply:
👍